File: LOO 2°Parcial/Vehiculos/src/clases/Remolque.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package clases;

/**
 *
 * @author Roberto
 */
public class Remolque 
{
    private double alto;
    private double ancho;
    private double largo;
    private double pesomax;
    public void setAlto(double alto)
    {
        this.alto = alto;
    }
    public double getAlto()
    {
        return(this.alto);
        
    }
    public void setAncho(double ancho)
    {
        this.ancho = ancho;
    }
    public double getAncho()
    {
        return(this.ancho);
        
    }
    public void setLargo(double largo)
    {
        this.largo = largo;
    }
    public double getLargo()
    {
        return(this.largo);
        
    }
    public void setPesomax(double pesomax)
    {
        this.pesomax = pesomax;
    }
    public double getPesomax()
    {
        return(this.pesomax);
        
    }
    
    public void poner()
    {
        System.out.println("Remolque instalado");
    }
    public void quitar()
    {
        System.out.println("Remolque desisntalado");
    }
    @Override
    public String toString()
    {
        String info = "Dimensiones: " + getAlto() + " de alto, " + getAncho() + " de ancho, ";
        info +=getLargo() + " m de largo.\n Peso maximo" + getPesomax() + "kg";
        return(info);
    }
}

File: LOO 2°Parcial/Simulador de Vehículos/Simulador de Vehículos/Remolque.cpp
#include "Remolque.h"
Remolque::Remolque() {
	this->alto = 0;
	this->ancho = 0;
	this->largo = 0;
	this->peso_max = 0;
}
Remolque::Remolque(double peso, double alto, double ancho, double largo) {
	this->alto = alto;
	this->ancho = ancho;
	this->largo = largo;
	this->peso_max = peso;
}
double Remolque::Get_alto() {
	return this->alto;
}
double Remolque::Get_ancho() {
	return this->ancho;
}
double Remolque::Get_largo() {
	return this->largo;
}
double Remolque::Get_pesomax() {
	return this->peso_max;
}
void Remolque::Set_alto(double alto) {
	this->alto = alto;
}
void Remolque::Set_ancho(double ancho) {
	this->ancho = ancho;
}
void Remolque::Set_largo(double largo) {
	this->largo = largo;
}
void Remolque::Set_pesomax(double peso) {
	this->peso_max = peso;
}
void Remolque::to_string(void) {
	cout << "Informacion del remolque: " << endl;
	cout << "Alto: " << this->alto << "\nAncho: " << this->ancho << "\nLargo: " << this->largo << "\nPeso maximo: " << this->peso_max << endl;
}
